src/status_im/chat/models.cljs

```diff
@@ -110,10 +110,12 @@
 
 (defn map-chats [{:keys [db] :as cofx}]
   (fn [val]
-    (merge
-     (or (get (:chats db) (:chat-id val))
-         (create-new-chat (:chat-id val) cofx))
-     val)))
+    (assoc
+     (merge
+      (or (get (:chats db) (:chat-id val))
+          (create-new-chat (:chat-id val) cofx))
+      val)
+     :invitation-admin (:invitation-admin val))))
```

Comment:
Looks like it is redundant as the whole `val` is merged above?

Reply:
not its not, this is for `nil`